README.md

```diff
@@ -128,16 +128,41 @@ To start Boulder in a Docker container, run:
 docker-compose up
 ```
 
-To run tests:
+To run all tests:
 
 ```shell
-docker-compose run --use-aliases boulder ./test.sh
+./test_dev.sh -a
```

Comment:
suggestion: `-a` shouldn't be necessary, running everything should be the assumed mode. Same for the unit/integration subcommands. Feel free to ignore if this is prohibitively difficult given the getopts mechanism.

Reply:
The `getopts` mechanism is still new to me, I'm sure I could figure out a way to implement what you're asking for though. I can definitely understand where you're coming from here. This decision was born out of my own experience try to SIGTERM a couple accidental runs and having it result in a hang. If you still think that it's too wonky, I am not opposed to adjusting the behavior of the script.

Reply:
FYI, if you SIGTERM a `docker-compose` run and it's hanging, sometimes Ctrl-\ will unstick it quickly. We really should figure out why those hangs happen though. I'd like Ctrl-C to cleanly and quickly end the tests.